PermissionsEx/permissions.yml

```diff
@@ -1113,6 +1113,8 @@ groups:
         permissions:
         - ch.alias.mr.shifty
         - -essentials.protect.damage.fall
+        - worldedit.navigation.thru.command
+        - worldedit.navigation.jumpto.command 
```

Comment:
he can only go as far as he can see, or for the game's sake, as far as the little dot is so he cant skip entire rooms.